website/content/docs/nia/enterprise/index.mdx

```diff
@@ -0,0 +1,14 @@
+---
+layout: docs
+page_title: Consul-Terraform-Sync Enterprise
+description: >-
+  Consul-Terraform-Sync Enterprise
+---
+
+# Consul-Terraform-Sync Enterprise
+
+Consul-Terraform-Sync Enterprise features address organization complexities of collaboration, operations, scale, and governance. Please see the sidebar navigation on the left to view all enterprise features.
+
+Enterprise features require a Consul [license](/docs/nia/enterprise/license) to be applied.
+
+These features are part of Consul-Terraform-Sync Enterprise which is part of [Consul Enterprise](https://www.hashicorp.com/products/consul).
```

Comment:
[Consul-ent](https://www.consul.io/docs/enterprise) and [vault-ent](https://www.vaultproject.io/docs/enterprise) both have this "These features are part of..." statement at the bottom of their enterprise overview page. Following the pattern there.